homeassistant/components/homekit/type_covers.py

```diff
@@ -184,7 +186,7 @@ def set_tilt(self, value):
 
         self.call_service(DOMAIN, SERVICE_SET_COVER_TILT_POSITION, params, value)
 
-    def update_state(self, new_state):
+    def async_update_state(self, new_state):
```

Comment:
Should this be decorated with `@callback`?

Reply:
Yes, will fix.